src/components/auth/ProfileModal/contents/evm-linking/LinkEvmAddressContent.tsx

```diff
@@ -35,20 +38,35 @@ function LinkEvmAddressContent({ evmAddress, setCurrentState }: ContentProps) {
     />
   )
 
+  const unlinkButton = (
+    <Button
+      onClick={() => setCurrentState('unlink-evm-confirmation')}
+      disabled={!canUnlinkAddress}
+      className='mt-4 w-full border-red-500'
+      variant='primaryOutline'
+      size='lg'
+    >
+      Unlink EVM address
+    </Button>
+  )
+
   return (
     <div>
       {evmAddress ? (
         <div>
           <EvmAddress evmAddress={evmAddress} className='mb-2' />
           {isNotEqAddresses && connectionButton}
-          <Button
-            onClick={() => setCurrentState('unlink-evm-confirmation')}
-            className='mt-4 w-full border-red-500'
-            variant='primaryOutline'
-            size='lg'
-          >
-            Unlink EVM address
-          </Button>
+          {!canUnlinkAddress ? (
+            <PopOver
+              yOffset={8}
+              trigger={<div className='w-full'>{unlinkButton}</div>}
+              triggerOnHover
+            >
+              <p>You need at least 1 identity/account linked to your account</p>
+            </PopOver>
+          ) : (
+            unlinkButton
```

Comment:
This one is copy-paste in EVM and Polkadot files

Reply:
refactored